packages/calcite-components/src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -496,7 +496,12 @@ export class InputTimePicker
     syncHiddenFormInput("time", this, input);
   }
 
-  private timeChangeHandler({ detail: newValue }): void {
+  private timeChangeHandler(event): void {
```

Comment:
Why not just change `this.listenOn(window, "calciteTimeChange", this.timeChangeHandler);` to not be the window?

Reply:
Great suggestion.  I remember the event not working at first, so I changed it to listen on the window, but it turns out that does actually work, so I'm going to update it to not be on the window after all.

Reply:
done!